packages/react-docs/RELEASE-NOTES.md

```diff
@@ -3,6 +3,125 @@ id: Release notes
+  - Added OUIA props ([#5949](https://github.com/patternfly/patternfly-react/pull/5949))
+- **Code editor:**
+  - Added support for more languages ([#6006](https://github.com/patternfly/patternfly-react/pull/6006))
+  - Added showEditor prop ([#6022](https://github.com/patternfly/patternfly-react/pull/6022))
+  - Updated to allow passing options and overrideServices ([#6035](https://github.com/patternfly/patternfly-react/pull/6035))
+  - Updated to show toolbar if there are custom controls ([#6025](https://github.com/patternfly/patternfly-react/pull/6025))
+- **Context selector:**
+  - Updated to hide internal prop ([#5986](https://github.com/patternfly/patternfly-react/pull/5986))
+- **Date picker:**
+  - Changed default parse format to y-m-d from y(-m)?(-d)? ([#6039](https://github.com/patternfly/patternfly-react/pull/6039))
+  - Cleared parse error on type ([#6066](https://github.com/patternfly/patternfly-react/pull/6066))
+- **Dual list selector:**
+  - Fixed button order ([#5983](https://github.com/patternfly/patternfly-react/pull/5983))
+- **Expandable section:**
+  - Added props to support disclosure variation ([#6078](https://github.com/patternfly/patternfly-react/pull/6078))
+- **Form:**
+  - Added form group label info ([#6053](https://github.com/patternfly/patternfly-react/pull/6053))
+- **Label:**
+  - Added editable label ([#6065](https://github.com/patternfly/patternfly-react/pull/6065))
+- **Menu:**
+  - Addded context menu flyout support ([#5789](https://github.com/patternfly/patternfly-react/pull/5789))
+- **Navigation:** Updated incorrect prop descriptions ([#5980](https://github.com/patternfly/patternfly-react/pull/5980))
+- **Notification drawer:**
+  -  Fixed to show close button when onClose provided ([#6071](https://github.com/patternfly/patternfly-react/pull/6071))
+- **Page:**
+  - Added tabs page section variant ([#6028](https://github.com/patternfly/patternfly-react/pull/6028))
+- **Pagination:**
+  - Exported interfaces in md file so prop descriptions are documented ([#5985](https://github.com/patternfly/patternfly-react/pull/5985))
+- **Search input:**
+  - Updated to allow passing custom form elements after the attributes ([#6043](https://github.com/patternfly/patternfly-react/pull/6043))
+- **Select:**
+  - Updated to check children have props before comparing them ([#6052](https://github.com/patternfly/patternfly-react/pull/6052))
+  - Updated to check the values of the children for possible changes on componentDidUpdate ([#5973](https://github.com/patternfly/patternfly-react/pull/5973))
+  - Fixed keyboard handling for view more button and footer ([#6095](https://github.com/patternfly/patternfly-react/pull/6095))
+  - Fixed to ignore arrow keys when typeahead options are empty ([#5958](https://github.com/patternfly/patternfly-react/pull/5958))
+  - Made type improvements on onFilter ([#6073](https://github.com/patternfly/patternfly-react/pull/6073))
+- **Search input:**
```

Comment:
1. This can be combined with the `Search input` note above `Select`.
2. Past tense - "Updated to allow hasWordAttrLabel..."